### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by michalc**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: python:3.7.1
+      - image: quay.io/uktrade/ftps-s3-minio:latest
+        environment:
+          MINIO_ACCESS_KEY: AKIAIOSFODNN7EXAMPLE
```
Comment (lgarvey) [inline .circleci/config.yml]: @michalc should these be here?

File: tests.py [hunk 2]
```diff
@@ -18,58 +27,310 @@ def wrapper(*args, **kwargs):
...
     @async_test
-    async def test_empty_bucket(self):
-        await self.setup_manual()
+    async def test_if_correct_creds_login_succeeds(self):
+        loop = await self.setup_manual()
+
+        def nothing(_):
+            pass
+
+        await ftp_run(nothing, loop=loop, user='my-user', passwd='my-password')
```
Comment (lgarvey) [inline tests.py]: assertNotRaises?

File: tests.py [hunk 3]
```diff
@@ -18,58 +27,310 @@ def wrapper(*args, **kwargs):
...
-        context = ssl.SSLContext()
-        async with aioftp.ClientSession('127.0.0.1', port=8021, user='user', password='anon@',
-                                        ssl=context) as client:
-            results = await client.list()
 
-        self.assertEqual(results, [])
+def env():
+    return {
+        'AWS_AUTH_MECHANISM': 'secret_access_key',
+        'AWS_ACCESS_KEY_ID': 'AKIAIOSFODNN7EXAMPLE',
```
Comment (lgarvey) [inline tests.py]: Should these be here?

Comment (lgarvey) [reply tests.py]: If they are fake, can you make that obvious?  e.g. 'an-aws-access-key' or something

File: tests.py (hunk 2 above)
Comment (michalc) [reply tests.py]: Ah... actually, I'm not sure there is an assertNotRaises? The test would fail if it does raise

**Pull request merged**